### PR TITLE
doc: relocate FOTA build file docs

### PR DESCRIPTION
**Pull request by plskeggs**
Move generic information about FOTA zip files to the app_build_system.rst file from the nrf_cloud.rst file.

Jira: NCSDK-12829

### CODE REVIEW

File: doc/nrf/app_build_system.rst [hunk 1]
```diff
@@ -43,3 +43,22 @@ You must be aware of these additions when you start writing your own application
 * The |NCS| build system extends Zephyr's with support for multi-image builds.
   You can find out more about these in the :ref:`ug_multi_image` section.
 * The |NCS| adds a partition manager, responsible for partitioning the available flash memory.
+* The |NCS| build system generates zip files containing binary images and a manifest for use with nRF Cloud FOTA.  See below.
```
Comment (peknis) [inline doc/nrf/app_build_system.rst:46]:
```suggestion
* The |NCS| build system generates zip files containing binary images and a manifest for use with nRF Cloud FOTA.
```

File: doc/nrf/app_build_system.rst [hunk 2]
```diff
@@ -43,3 +43,22 @@ You must be aware of these additions when you start writing your own application
...
+
+.. _app_build_fota:
+
+Building FOTA images
+********************
+
+The |NCS| build system places output images in the :file:`<build folder>/zephyr` folder.
+
+If :kconfig:`CONFIG_BOOTLOADER_MCUBOOT` is set, the build system creates the :file:`dfu_application.zip` file containing files :file:`app_update.bin` and :file:`manifest.json`.
```
Comment (peknis) [inline doc/nrf/app_build_system.rst:55]:
```suggestion
If you have set the :kconfig:`CONFIG_BOOTLOADER_MCUBOOT` option, the build system creates the :file:`dfu_application.zip` file containing files :file:`app_update.bin` and :file:`manifest.json`.
```

File: doc/nrf/app_build_system.rst [hunk 3]
```diff
@@ -43,3 +43,22 @@ You must be aware of these additions when you start writing your own application
...
+Further, if :kconfig:`CONFIG_IMG_MANAGER` and :kconfig:`CONFIG_MCUBOOT_IMG_MANAGER` are also set, then the application will be able to process FOTA updates.
```
Comment (peknis) [inline doc/nrf/app_build_system.rst:56]:
```suggestion
If you have also set the options :kconfig:`CONFIG_IMG_MANAGER` and :kconfig:`CONFIG_MCUBOOT_IMG_MANAGER`, the application will be able to process FOTA updates.
```

File: doc/nrf/app_build_system.rst [hunk 4]
```diff
@@ -43,3 +43,22 @@ You must be aware of these additions when you start writing your own application
...
+If :kconfig:`CONFIG_SECURE_BOOT` and :kconfig:`CONFIG_BUILD_S1_VARIANT` are set, a similar file :file:`dfu_mcuboot.zip` will also be created.
+This file can be used to perform FOTA updates of MCUboot itself.
```
Comment (peknis) [inline doc/nrf/app_build_system.rst:58]:
```suggestion
If you have set the options :kconfig:`CONFIG_SECURE_BOOT` and :kconfig:`CONFIG_BUILD_S1_VARIANT`, a similar file :file:`dfu_mcuboot.zip` will also be created.
You can use this file to perform FOTA updates of MCUboot itself.
```

File: doc/nrf/app_build_system.rst [hunk 5]
```diff
@@ -43,3 +43,22 @@ You must be aware of these additions when you start writing your own application
...
+
+The :file:`app_update.bin` file is a signed version of your application.
+The signature matches to what MCUboot expects, so it allows this file to be used as an update.
```
Comment (peknis) [inline doc/nrf/app_build_system.rst:61]:
```suggestion
The signature matches to what MCUboot expects and allows this file to be used as an update.
```

File: doc/nrf/app_build_system.rst [hunk 6]
```diff
@@ -43,3 +43,22 @@ You must be aware of these additions when you start writing your own application
...
+The build system creates a :file:`manifest.json` file using information in the :file:`zephyr.meta` output file.
```
Comment (peknis) [inline doc/nrf/app_build_system.rst:62]:
```suggestion
The build system creates a :file:`manifest.json` file using the information in the :file:`zephyr.meta` output file.
```

File: doc/nrf/libraries/networking/nrf_cloud.rst [hunk 7]
```diff
@@ -111,7 +111,17 @@ Configuration options for device ID
 Firmware over-the-air (FOTA) updates
 ************************************
 The nRF Cloud library supports FOTA updates for your nRF9160-based device.
-When the library is included by the application, the :kconfig:`CONFIG_NRF_CLOUD_FOTA` option is enabled by default, and the FOTA functionality is made available to the application.
+The :kconfig:`CONFIG_NRF_CLOUD_FOTA` option is enabled by default when :kconfig:`CONFIG_NRF_CLOUD_MQTT` is set.
+This enables FOTA functionality in the application.
+
+In order to do its job, nRF Cloud FOTA enables additional features and libraries:
```
Comment (peknis) [inline doc/nrf/libraries/networking/nrf_cloud.rst:117]:
```suggestion
nRF Cloud FOTA enables the following additional features and libraries:
```

File: doc/nrf/libraries/networking/nrf_cloud.rst [hunk 8]
```diff
@@ -147,18 +157,10 @@ If the value equals :c:enumerator:`NRF_CLOUD_FOTA_MODEM`, the application can op
...
-If :kconfig:`CONFIG_BOOTLOADER_MCUBOOT` is set, the build system creates the file :file:`dfu_application.zip` file containing files :file:`app_update.bin` and :file:`manifest.json`.
-Further, if :kconfig:`CONFIG_IMG_MANAGER` and :kconfig:`CONFIG_MCUBOOT_IMG_MANAGER` are also set, then the application will be able to process FOTA updates.
-
-The :file:`app_update.bin` file is a signed version of your application.
-The signature matches to what MCUboot expects, so it allows this file to be used as an update.
-The build system creates a :file:`manifest.json` file using information in the :file:`zephyr.meta` output file.
-This includes the Zephyr and |NCS| git hashes for the commits used to build the application.
-If your working tree contains uncommitted changes, the build system adds the suffix ``-dirty`` to the relevant version field.
+The build system will create the files :file:`dfu_application.zip` and/or :file:`dfu_mcuboot.zip` for a properly configured application.
+See :ref:`app_build_system` for more information about FOTA zip files.
```
Comment (peknis) [inline doc/nrf/libraries/networking/nrf_cloud.rst:161]: Shouldn't the link target to app_build_fota instead?

**Review by peknis — CHANGES_REQUESTED** [review]

**Comment by greg-fer** [comment]
Please check the changelog label. If an entry is not needed, remove the label. Thanks.

**Comment by peknis** [comment]
> Please check the changelog label. If an entry is not needed, remove the label. Thanks.

No changelog entry needed as it is in the proper place already. The actual information was in wrong place and this PR moves it to the proper place.

File: doc/nrf/libraries/networking/nrf_cloud.rst (hunk 8 above)
Comment (plskeggs) [reply doc/nrf/libraries/networking/nrf_cloud.rst:161]: yes

**Review by jayteemo — APPROVED** [review]

**Comment by plskeggs** [comment]
@peknis I made all your changes.

**Review by peknis — APPROVED** [review]

**Pull request merged**